kyuubi-relocated-spark-connect-rpc/src/main/protobuf/README.md

```diff
@@ -0,0 +1,8 @@
+proto files are copied from
+https://github.com/apache/spark/tree/5b2d2149b615acdd8730547a1f24c2b637222545/sql/connect/common/src/main/protobuf
+
+and with one additional change in each proto file
```

Comment:
We can provide an automatic script or command for batch modification in the future.

Reply:
The cost of manual modification is not high, but we do need CI to ensure the correctness of the package name